package.json

```diff
@@ -31,24 +31,43 @@
     "@angular/platform-browser": "^8.2.0",
     "@angular/platform-browser-dynamic": "^8.2.0",
     "@angular/router": "^8.2.0",
+    "@angular-devkit/build-angular": ">= 0.800.0 < 0.900.0",
     "@angular-devkit/build-ng-packagr": "~0.803.29",
     "@commitlint/cli": "^12.0.0",
     "@commitlint/config-conventional": "^12.1.1",
     "@compodoc/compodoc": "~1.1.11",
     "@nationalbankbelgium/code-style": "^1.5.0",
     "@ngtools/json-schema": "^1.1.0",
+    "@npmcli/ci-detect": "^1.3.0",
     "@release-it/conventional-changelog": "^2.0.1",
+    "@types/jasmine": "~3.6.4",
+    "@types/karma": "~5.0.1",
+    "@types/node": "^10.17.54",
     "codelyzer": "^5.2.2",
     "commitizen": "^4.2.3",
     "conventional-changelog-cli": "^2.1.1",
     "core-js": "^3.9.1",
+    "coveralls": "^3.1.0",
     "cross-env": "^7.0.3",
     "cz-customizable": "^6.3.0",
     "hammerjs": "^2.0.8",
     "husky": "^4.3.8",
+    "jasmine-core": "^3.6.0",
+    "karma": "^5.2.3",
```

Comment:
About moving all deps from `stark-testing` to the root... could we try something else so we can avoid duplication?

What about just adding `stark-testing` as DevDependency here and link it to the stark-testing folder (no need for a compiled `tgz` I think because we don't compile anything in that package). That way the deps will be installed at the root... right? :thinking:

Reply:
Ha ha, I didn't think to this since the dependencies are not well maintained after (doing `npm i` from root after upgrading deps in stark-testing has no impact on installed deps in root).

However, since we are looking for a better solution in the future to manage our monorepo, I think it's better to use your solution and not copy deps from testing into root folder 👍